CrewChiefV4/UserInterface/MainWindow.cs

```diff
@@ -697,14 +697,28 @@ public MainWindow()
             this.filenameLabel.Visible = CrewChief.Debugging;
             this.filenameTextbox.Visible = CrewChief.Debugging;
             this.playbackInterval.Visible = CrewChief.Debugging;
+           
+            String[] commandLineArgs = Environment.GetCommandLineArgs();
+            if (commandLineArgs != null)
+            {
+                foreach (String arg in commandLineArgs)
+                {
+                    if (arg.Equals("SMOKE_TEST"))
```

Comment:
as this is a command line parameter that may be added by a user (sound pack creator), perhaps the term "SMOKE_TEST" isn't really meaningful. How about "SOUND_TEST"?